addons/website_forum/views/forum_forum_views.xml

```diff
@@ -15,6 +15,7 @@
                     <field name="total_answers" optional="hide"/>
                     <field name="total_favorites" optional="hide"/>
                     <field name="active" column_invisible="True"/>
+                    <field name="sequence" column_invisible="True"/>
```

Comment:
This line should not be necessary we already have `<field name="sequence" widget="handle"/>` above, otherwise it looks good to me. 

I will also take into account for the client and the fix that if they mark the "Separate Courses" check then the sorting will be different and can be confusing since it will keep the order but it will separate the forums in 2 "blocks" one with the ones that does not belong to a course and the course discussions, you could double check, but for me it seems that the sorting works fine in runbot? maybe it's still a good idea to prioritize the sequence if it's what we want.

PS: Not very important but there's a small typo in the commit message liste -> list

Thanks 😊!

Reply:
Indeed, I did not understand what you meant about the separate courses? The sorting works fine on runbot.
Will change the typo thanks :)